src/DacpacTool/Program.cs

```diff
@@ -97,6 +99,23 @@ private static int BuildDacpac(BuildOptions options)
                 }
             }
 
+            // Add input files by iterating through $Project.InputFiles.txt
+            if (options.InputFile != null)
+            {
+                if (File.Exists(options.InputFile))
+                {
+                    foreach (var line in File.ReadLines(options.InputFile))
+                    {
+                        FileInfo inputFile = new FileInfo(line); // Validation occurs in AddInputFile
+                        packageBuilder.AddInputFile(inputFile);
+                    }
+                }
+                else 
```

Comment:
This path shouldn't be possible, since `options.InputFile` is created by the SDK, but I wanted to make sure it was handled just in case.

Reply:
Yeah, I think it is a good idea in case some one does want to invoke the tool manually for some reason.